src/components/atoms/ImagePicker/ImagePicker.component.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { Upload as AntUpload, UploadProps as AntUploadProps } from 'antd'
+
+import { ComponentProps } from '@src/types/component'
+
+import { PreviewProps } from '@src/types/preview'
+
+import VisibilityComponent from '@components/common/ResponsiveVisibility/ResponsiveVisibility.component'
+
+export interface ImagePickerProps
+  extends PreviewProps,
+    ComponentProps<Omit<AntUploadProps, 'listType'>> {}
+
+const ImagePicker = React.forwardRef<HTMLDivElement, ImagePickerProps>((props, ref) => {
+  const { children, isPreview, responsiveVisibility, ...rest } = props
+
+  return (
+    <VisibilityComponent visibility={responsiveVisibility} isPreview={isPreview}>
+      <div ref={ref}>
+        <AntUpload {...rest} disabled={isPreview} listType="picture">
```

Comment:
should we keep `Upload` or renaming to `ImagePicker` if renaming, should we limit extensions are allowed?

Reply:
@quy-truong Absolutely, we should limit the extension. Let me improve it. Thanks for the suggestion

Reply:
```
<AntUpload  listType="picture" disabled={isPreview} {...rest} >
```
just let user over write it in case they want to update